src/api/photo-center/index.ts

```diff
@@ -23,6 +24,19 @@ export class ArcProtoCenter extends ArcAbstractAPI {
     return data;
   }
 
+  async uploadImageStream(readableStream: ReadStream){
+    const form = new FormData();
+
+    form.append('file', readableStream, {
+      filename: 'file',
```

Comment:
filename: 'file' – Should come from the stream (readableStream.path) or be passed as an argument.
contentType: 'application/json' – This is likely wrong for images


```suggestion
    const filename = options?.filename ?? (readableStream.path ? String(readableStream.path).split('/').pop() : 'file');
    const contentType = options?.contentType ?? 'application/octet-stream';

    form.append('file', readableStream, { filename, contentType });
```